src/components/CheckList/CheckListHeader.tsx

```diff
@@ -57,14 +54,6 @@ export const CheckListHeader = ({
   return (
     <>
       <div className={styles.row}>
-        <div>
-          {!scenesEnabled ||
-            (viewType !== CheckListViewType.Viz && (
-              <div>
-                Currently showing {currentPageChecks.length} of {checks.length} total checks
-              </div>
-            ))}
-        </div>
```

Comment:
Do we want to remove this? I find this information quite useful because of the filters / pagination 😄

Reply:
Oopsy doopsy